interface/main/calendar/modules/PostCalendar/pntemplates/compiled/README

```diff
@@ -1,2 +0,0 @@
-This is a dummy file to ensure this directory is included in software
-package release.
```

Comment:
I confirmed on testing that this does break the install.
(ie. add this file back :) )